components/blitz/src/pojos/ShapeSettingsData.java

```diff
@@ -161,7 +161,9 @@ public Color getFill()
 		Shape shape = (Shape) asIObject();
 		RInt value = shape.getFillColor();
 		if (value == null) return DEFAULT_FILL_COLOUR;
-		return new Color(value.getValue(), true);
+		Color c = new Color(value.getValue(), true);
```

Comment:
NB: seems like this might be a candidate for either a change to the existing `Color` constructor or a new one with another boolean flag `lenient`, etc.